### PR TITLE
Customizable color warning

### PR DESCRIPTION
**Pull request by NestorEncinas**
JIRA: ONE-4649

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1533/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3109/ : SUCCESS in 10m 35s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1534/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3110/ : SUCCESS in 10m 44s

File: libs/sdk-ui-kit/styles/scss/variables.scss [hunk 1]
```diff
@@ -43,9 +43,19 @@ $gd-palette-primary-base-darken06: var(--gd-palette-primary-base-d06, darken($gd
...
-$gd-color-warning-label: mix($gd-color-warning, $gd-color-text, 20%);
+
+// warning colors
+$gd-palette-warning-base: var(--gd-palette-warning-base, $gd-color-warning);
+$gd-palette-warning-dimmed: var(
+    --gd-palette-warning-dimmed,
+    mix($gd-color-warning, $gd-color-text-light, 20%)
+);
+$gd-palette-warning-base-t50: var(--gd-palette-warning-base-t50, transparentize($gd-color-warning, 0.5));
+$gd-color-warning-label: var(--gd-palette-warning-dimmed, mix($gd-color-warning, $gd-color-text, 20%));
```
Comment (ivan-nejezchleb) [inline libs/sdk-ui-kit/styles/scss/variables.scss:54]: this is using the same --gd value, but different default:
mix($gd-color-warning, $gd-color-text-light, 20%) VS
mix($gd-color-warning, $gd-color-text, 20%)
so it should have probably different --gd variable filled differently in TS code

**Review by ivan-nejezchleb — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by ivan-nejezchleb — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/499/ : SUCCESS in 23s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/759/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3115/ : SUCCESS in 11m 32s

**Pull request merged**